clients/src/main/java/org/apache/kafka/clients/consumer/internals/PartitionAssignor.java

```diff
@@ -243,10 +239,18 @@ Short version() {
             return partitions;
         }
 
-        public ConsumerProtocol.Errors error() {
+        public ConsumerProtocol.AssignmentError error() {
             return error;
         }
 
+        public void updatePartitions(List<TopicPartition> partitions) {
```

Comment:
Do these methods have to be public?

Reply:
Ah good catch, it needs to be default-access.